backend/ToDosBackend/src/main/resources/application.properties

```diff
@@ -1 +1,2 @@
 spring.application.name=ToDosBackend
```

Comment:
All good but probably best to use application.yml instead of application.properties